domains/authorization/Authorization.API/API/Authorization/Exceptions/ExceptionHandlerMiddleware.cs

```diff
@@ -45,4 +45,16 @@ public async Task Invoke(HttpContext context)
     }
 }
 
-public record Error(HttpStatusCode StatusCode, String Description);
+public class Error
+{
+    public string Status { get; private set; }
+    public int StatusCode { get; private set; }
+    public string Description { get; private set; }
+
+    public Error(HttpStatusCode statusCode, string description)
+    {
+        Status = statusCode.ToString();
+        StatusCode = (int)statusCode;
```

Comment:
Will this result in the status code being returned as a string and an int? 
eg.
Status = "500"
StatusCode = 500

Reply:
Maybe a test to check this error?

Reply:
"StatusCode": 500,
"Status": "Internal server error"